src/test/versionbits_tests.cpp

```diff
@@ -314,7 +312,7 @@ static void check_computeblockversion(const Consensus::Params& params, Consensus
     BOOST_REQUIRE(0 <= bit && bit < 32);
     BOOST_REQUIRE(((1 << bit) & VERSIONBITS_TOP_MASK) == 0);
     BOOST_REQUIRE(min_activation_height >= 0);
-    BOOST_REQUIRE_EQUAL(min_activation_height % params.nMinerConfirmationWindow, 0);
+    BOOST_REQUIRE_EQUAL(min_activation_height % params.nMinerConfirmationWindow, 0U);
```

Comment:
Seems like there's a few of these issues: https://github.com/bitcoin/bitcoin/pull/21701#pullrequestreview-637244766

Reply:
The other instance is removed in the next commit